File: src/FrontEnd/desafiodev-app/src/models/Transaction.js
// Transaction.js

class Transaction {
  id;
  type;
  date;
  value;
  cpf;
  card; 
  time;
  storeOwner;
  storeName;

  constructor({
    id,
    type,
    date,
    value,
    cpf,
    card,
    time,
    storeOwner,
    storeName
  }) {
    this.id = id;
    this.type = type;
    this.date = date;
    this.value = value;
    this.cpf = cpf;
    this.card = card;
    this.time = time;
    this.storeOwner = storeOwner;
    this.storeName = storeName;
  }
}

export default Transaction;

File: src/Backend/DesafioDevApi/Domain/Entities/Transaction.cs
namespace DesafioDevApi.Domain.Entities;

public class Transaction
{
    public int Id { get; set; }
    public int Type { get; set; }
    public DateTime Date { get; set; }
    public decimal Value { get; set; }
    public string CPF { get; set; }
    public string Card { get; set; }
    public DateTime Time { get; set; }
    public string StoreOwner { get; set; }
    public string StoreName { get; set; }
}
